### ISSUE TITLE
Impact of Something Something and Kinetics during Unlabeled Pre-training

### ISSUE THREAD

**Issue by edessa** [OpenGVLab/VideoMAEv2#56]
Hello,

In your unlabeled pre-training phase, you train on two datasets (STV2 and Kinetics) with reconstruction loss and later fine-tune with loss computed against their labels. Did you find that training over these two datasets during the unsupervised stage helped when finetuning? Did you find training over the validation sets to help as well? Thank you ahead of time, and great work on this paper!
